.code-samples.meilisearch.yaml

```
@@ -216,16 +234,12 @@ create_a_key_1: |-
       }
   }
 update_a_key_1: |-
-  let keyParams = KeyParams(
-    description: "Add documents: Products API key",
-    actions: ["documents.add", "documents.delete"],
-    indexes: ["products", "reviews"],
-    expiresAt: "2042-04-02T00:42:42Z"
+  let keyParams = KeyUpdateParams(
+    description: "Manage documents: Products/Reviews API key",
+    name: "Products/Reviews API key"
```

```suggestion
    name: "Products/Reviews API key",
    description: "Manage documents: Products/Reviews API key"
```

Just for consistency with the other samples of the other languages